src/include/uapi/abi.h

```diff
@@ -51,7 +51,7 @@
 #define __INCLUDE_UAPI_ABI_H__
 
 /** \brief SOF ABI version major, minor and patch numbers */
-#define SOF_ABI_MAJOR 3
+#define SOF_ABI_MAJOR 4
 #define SOF_ABI_MINOR 2
```

Comment:
Shouldn't you reset minor to 0 here? I thought incrementing major would reset all less significant version components

Reply:
```c
 * 1) Increment MAJOR version if you make incompatible API changes. MINOR and
 *    PATCH should be reset to 0.
```